packages/client/hmi-client/src/components/model/petrinet/tera-parameter-entry.vue

```diff
@@ -15,71 +15,82 @@
-						:model-value="getParameterDistribution(modelConfiguration, parameterId)?.parameters.maximum"
+						v-if="getParameterDistribution(modelConfiguration, parameterId).type === DistributionType.Constant"
+						label="Constant"
+						:model-value="getParameterDistribution(modelConfiguration, parameterId)?.parameters.value"
 						@update:model-value="
 							emit('update-parameter', {
 								id: parameterId,
-								distribution: formatPayloadFromParameterChange({ maximum: $event })
+								distribution: formatPayloadFromParameterChange({ value: $event })
 							})
 						"
 					/>
-				</template>
+					<!-- Uniform Distribution -->
+					<template v-if="getParameterDistribution(modelConfiguration, parameterId).type === DistributionType.Uniform">
+						<tera-input-number
+							label="Min"
+							:model-value="getParameterDistribution(modelConfiguration, parameterId)?.parameters.minimum"
+							@update:model-value="
+								emit('update-parameter', {
+									id: parameterId,
+									distribution: formatPayloadFromParameterChange({ minimum: $event })
+								})
+							"
+						/>
+						<tera-input-number
+							label="Max"
+							:model-value="getParameterDistribution(modelConfiguration, parameterId)?.parameters.maximum"
+							@update:model-value="
+								emit('update-parameter', {
+									id: parameterId,
+									distribution: formatPayloadFromParameterChange({ maximum: $event })
+								})
+							"
+						/>
+					</template>
+				</span>
+				<section>
+					<Button :label="getSourceLabel(parameterId)" text size="small" @click="isSourceOpen = !isSourceOpen" />
+					<Button :label="getOtherValuesLabel" text size="small" @click="showOtherConfigValueModal = true" />
+				</section>
+			</main>
+			<footer v-if="isSourceOpen" class="mb-2">
+				<tera-input-text
+					placeholder="Add a source"
+					:model-value="getParameterSource(modelConfiguration, parameterId)"
+					@update:model-value="emit('update-source', { id: parameterId, value: $event })"
+				/>
+			</footer>
+		</template>
+		<div v-else-if="inferredDistribution" class="inferred-parameter">
+			<span class="type"><label>Type</label> {{ inferredDistribution.type }}</span>
+			<span class="mean">
+				<label>Mean</label> {{ displayNumber(inferredDistribution?.parameters?.mean.toString()) }}
 			</span>
-			<section>
-				<Button :label="getSourceLabel(parameterId)" text size="small" @click="isSourceOpen = !isSourceOpen" />
-				<Button :label="getOtherValuesLabel" text size="small" @click="showOtherConfigValueModal = true" />
-			</section>
-		</main>
-		<footer v-if="isSourceOpen" class="mb-2">
-			<tera-input-text
-				placeholder="Add a source"
-				:model-value="getParameterSource(modelConfiguration, parameterId)"
-				@update:model-value="emit('update-source', { id: parameterId, value: $event })"
-			/>
-		</footer>
+			<span class="std">
+				<label>STD</label> {{ displayNumber(inferredDistribution?.parameters?.stddev.toString()) }}
```

Comment:
Typically standard deviation is abbreviated "stddev" or "sd" .... std is something else.